geode-core/src/test/java/org/apache/geode/test/junit/rules/MemberStarterRule.java

```diff
@@ -59,7 +59,7 @@
 
   protected transient TemporaryFolder temporaryFolder;
   protected File workingDir;
-  protected int memberPort = -1;
+  protected int memberPort = 0;
```

Comment:
this change is not necessary, right?

Reply:
It is.

Reply:
when I change it back to -1, the test you added still passes.

Reply:
I should have mentioned that other tests fail if it stays as `-1`.